assets/admin/package.json

```diff
@@ -2,12 +2,17 @@
     "license": "MIT",
     "repository": "https://github.com/sulu/sulu-minimal.git",
     "scripts": {
-        "clean": "rimraf web/admin/build",
+        "clean": "rimraf ./../../web/admin/build",
         "build": "npm run clean && webpack --mode production",
-        "watch": "npm run clean && webpack --mode development -w"
+        "watch": "npm run clean && webpack --mode development -w",
+        "preinstall": "npm install symlink-dir --save-dev && symlink-dir ../../vendor vendor",
```

Comment:
this avoids that dependencies of sulu-* are installed in `node_modules/sulu-*/node-modules` as now npm think. this directory is a parent of the module so it can install sulu-* dependency also in `node_modules/*`.